### PR TITLE
doc/tools: refine tcmu-runner.md

### PR DESCRIPTION
**Pull request by zdover23**
This PR refines the English in tcmu-runner.md.

Signed-off-by: Zac Dover <zac.dover@gmail.com>

### CODE REVIEW

File: docs/tools/tcmu-runner.md [hunk 1]
```diff
@@ -19,29 +20,28 @@ Linux&reg; kernel. LIO supports all prevalent storage fabrics, including Fibre
...
+between the kernel SCSI target infrastructure and a userspace application.
+The kernel-level module that is involved is called `target_core_user` and can
+be viewed as a virtual HBA.
 
-*tcmu-runner* implements the userspace level side of the ank processing,
-handling the details of the TCMU interface (UIO, netlink, pthreads, and DBus).
-*tcmu-runner* exports a more simple C plugin API allowing the creation of
-*file handlers* to emulate various device types. This organization is shown
-in the figure below.
+*tcmu-runner* implements the userspace-level side of the ank processing and
```
Comment (damien-lemoal) [inline docs/tools/tcmu-runner.md:27]: There is a bug here... s/ank/SCSI command
Not sure how that got here :)

File: docs/tools/tcmu-runner.md [hunk 2]
```diff
@@ -50,10 +50,10 @@ target="_blank">GitHub</a>.  The project <a href="https://github.com/open-iscsi/
 target="_blank">*README*</a> file provides detailed information on how to
 compile, install and execute *tcmu-runner* daemon.
 
-The control of *tcmu-runner* emulated devices is achieved using the *targetcli*
-utility available as a package with most distributions. For instance, on
+The *targetcli* utility, which is available as a package in most distributions,
+controls devices that have been emulated with *tcmu-runner*. For example, on
```
Comment (damien-lemoal) [inline docs/tools/tcmu-runner.md:54]: that have been emulated -> that are emulated

Comment (zdover23) [reply docs/tools/tcmu-runner.md:54]: Repaired.

**Comment by damien-lemoal** [comment]
Applied. Thanks !

**Comment by zdover23** [comment]
Damn are you fast.

**Pull request merged**